spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/SpatialConnect.java

```diff
@@ -132,12 +165,37 @@ public void restartAllServices() {
         }
     }
 
+    public Observable<SCServiceStatusEvent> serviceStarted(final String serviceId) {
+        return Observable.create(new Observable.OnSubscribe<SCServiceStatusEvent>() {
+            @Override
+            public void call(final Subscriber<? super SCServiceStatusEvent> subscriber) {
+                SCService service = getServiceById(serviceId);
+                if (service != null && service.getStatus() == SCServiceStatus.SC_SERVICE_RUNNING) {
+                    subscriber.onNext(new SCServiceStatusEvent(SCServiceStatus.SC_SERVICE_STARTED));
+                    subscriber.onCompleted();
+                } else {
+                    serviceEvents.autoConnect()
+                            .subscribe(new Action1<SCServiceStatusEvent>() {
+                                @Override
+                                public void call(SCServiceStatusEvent event) {
+                                    if (event.getServiceId().equals(serviceId) &&
+                                            event.getStatus().equals(SCServiceStatus.SC_SERVICE_STARTED)) {
+                                        subscriber.onNext(event);
```

Comment:
Should we also call onCompleted() here?  I think we want to terminate this observable after the service starts